koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -33,6 +33,7 @@
 import signal
 
 import koji
+import koji.plugin
```

Comment:
Let's add a comment explaining this import (https://pagure.io/koji/pull-request/693)